src/Form/CornellMixpanelAnalyticsSettings.php

```diff
@@ -150,18 +167,11 @@ public function submitForm(array &$form, FormStateInterface $form_state) {
    * Validation function for the admin settings form.
    */
   public function validateForm(array &$form, FormStateInterface $form_state) {
-    // Check if we are live on Pantheon
-    if (isset($_ENV['PANTHEON_ENVIRONMENT']) && $_ENV['PANTHEON_ENVIRONMENT'] === 'live') {
-      // On live environment
-      if (empty($form_state->getValue('cornell_mixpanel_token'))) {
-        $form_state->setErrorByName('cornell_mixpanel_token', $this->t('You must enter a Production mixpanel token.'));
-      }
+    if (empty($form_state->getValue('cornell_mixpanel_token'))) {
+      $form_state->setErrorByName('cornell_mixpanel_token', $this->t('You must enter a Production mixpanel token.'));
     }
-    else {
-      // On dev environment
-      if (empty($form_state->getValue('cornell_mixpanel_test_token'))) {
-        $form_state->setErrorByName('cornell_mixpanel_test_token', $this->t('You must enter a Test mixpanel token.'));
-      }
+    if (empty($form_state->getValue('cornell_mixpanel_test_token'))) {
+      $form_state->setErrorByName('cornell_mixpanel_test_token', $this->t('You must enter a Test mixpanel token.'));
```

Comment:
```suggestion
      $form_state->setErrorByName('cornell_mixpanel_test_token', $this->t('You must enter a Mixpanel Test Token.'));
```

(idk if "Test Token" should be capitalized, but it was capitalized one other place, so I'm writing it that way for now)